labman/db/process.py

```diff
@@ -3090,23 +3090,66 @@ def generate_prep_information(self):
                 ('r', 'water_lot_id', 'water_lot'),
             ]
             sql = """
-                SELECT study_id, sample_id, content, run_name, experiment,
-                       fwd_cycles, rev_cycles, principal_investigator,
-                       et.description as sequencer_description,
-                       lpp.epmotion_robot_id as lepmotion_robot_id,
-                       epmotion_tm300_8_tool_id, epmotion_tm50_8_tool_id,
-                       master_mix_id, water_lot_id,
-                       gep.epmotion_robot_id as gepmotion_robot_id,
-                       epmotion_tool_id, kingfisher_robot_id,
-                       extraction_kit_id,
-                       p1.external_id as plate, w1.row_num as row_num,
-                       w1.col_num as col_num,
-                       p2.external_id as primer_composition,
-                       psc.barcode_seq as primer_set_composition,
-                       run_name as run_prefix, sp.sequencer_id as platform_id,
-                       sp.experiment as center_project_name
+                SELECT study_id, sample_id,
+                    -- BARCODE
+                    psc.barcode_seq AS barcode,
+                    -- primer_set_id links to marker_gene_primer_set_id,
+                    -- where we can get the linker/primer
+                    primer_set_id,
+                    -- primer_plate
+                    p2.external_id AS primer_plate,
+                    -- the well_id is a combination of the
+                    --- row/col_num + content
+                    w1.row_num AS row_num, w1.col_num AS col_num, content,
+                    -- where to get the platting name
+                    process.run_personnel_id AS plating,
+                    -- extractionkit_lot
+                    extraction_kit_id,
+                    -- extraction_robot, both the epmotion robot and the
+                    -- kingfisher robot
+                    lpp.epmotion_robot_id AS lepmotion_robot_id,
+                    kingfisher_robot_id,
+                    -- TM1000_8_tool
+                    epmotion_tool_id,
+                    -- mAStermix_lot
+                    mASter_mix_id,
+                    -- water_lot
+                    water_lot_id,
+                    -- processing_robot
+                    gep.epmotion_robot_id AS gepmotion_robot_id,
+                    -- TM300_8_tool
+                    epmotion_tm300_8_tool_id,
+                    -- TM50_8_tool
+                    epmotion_tm50_8_tool_id,
+                    -- sample_plate
+                    p1.external_id AS sample_plate,
+                    -- project_name
+                    study_alias AS project_name,
+                    -- orig_name
+                    sample_id AS orig_name,
+                    -- experiment_design_description
+                    '' AS experiment_design_description,
```

Comment:
The design description is always empty?

Reply:
...same question for some of those below

Reply:
Actually, this one is a mistake on my part--thanks for noticing it!  I classed this in the "leave empty" category (see below) because it is empty in the known-good example I have, but going back to my notes, I see that Gail suggested that we should pull it from the Qiita study description. 

Gail instructed us to always leave the following fields empty:
RUN_DATE
RUN_PREFIX
center_project_name
INSTRUMENT_MODEL
RUNID

I assume that someday in the future, labman or qiita or something will be expanded to close the loop by filling in these values after the sequencing is done ...

Reply:
Okay, thanks! Some of this is out-of-scope for labman too (e.g., the actual sequencing date)